pkg/plugins/platform/local.go

```diff
@@ -667,7 +661,7 @@ func (l *LocalApp) DockerEnv() {
 			// @ TODO: I have no idea what a Setenv error would even look like, so I'm not sure what
```

Comment:
nit, we can probably lose this TODO. I think we're doing all that's reasonable to be done here.

Reply:
Hey we've had that in there so long it's an old friend!